packages/text/src/Text.ts

```diff
@@ -383,7 +401,7 @@ export class Text extends Sprite
      * @param {PIXI.Rectangle} rect - The output rectangle.
      * @return {PIXI.Rectangle} The bounds.
      */
-    getLocalBounds(rect)
+    getLocalBounds(rect: Rectangle): Rectangle
```

Comment:
'public' modifier is required!

empty modifier only for 'internal' (not public API, but used outside of type, such as inside Renderer )

Reply:
Ah okay. I missed that there is a semantic difference for this, I'll check the visibility for all the methods.